.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+name: Release
+run-name: "Continous Release"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * 3' # this means every Wednesday @1am UTC
+
+jobs:
+
+  rc:
+    name: Check if we need a new release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      issues: read
+    outputs:
+      rc: ${{ steps.rc.outputs.new_release_published }}
+      ver: ${{ steps.rc.outputs.new_release_version }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Check if new version will be produced
+        id: rc
+        uses: cycjimmy/semantic-release-action@v3
+        with:
+          dry_run: true
```

Comment:
Shall we explicitly set the `ci: true` as well? It says the following in the semantic release action github page.

> ci can be used, e.g in combination with dry_run when generating the next release version in pull requests, where semantic_release would normally block the execution.

Reply:
nope, as this action discovers if it runs inside CI. For commonly used CIs this works pretty well (we use it for Terraform in the same way). You might want to use `ci: true` if you run it in some exotic environment. 

Github sets an env variable, `GITHUB_ACTION` or something similar, that is being discovered by semantic release action to check if this is a CI env.